lib/embedded_ansible/docker_embedded_ansible.rb

```diff
@@ -0,0 +1,200 @@
+class DockerEmbeddedAnsible < EmbeddedAnsible
+  AWX_WEB_PORT = "54321".freeze
+
+  def self.available?
+    require 'docker'
+    Docker.validate_version!
+  rescue RuntimeError
+    false
+  end
+
+  def initialize
+    super
+    require 'docker'
+  end
+
+  def start
+    run_rabbitmq_container
+    run_memcached_container
+    sleep(15)
```

Comment:
I actually yanked this directly from the [playbook in ansible/awx](https://github.com/ansible/awx/blob/devel/installer/local_docker/tasks/main.yml#L118-L121)

Because these are not "services" it seems harder to check their status.
Maybe we can pull the `ncat` checks into ManageIQ/manageiq in some way and reuse those here and in the pod in a follow up?